docs/backend-system/building-backends/08-migrating.md

```diff
@@ -754,3 +754,255 @@ going easily, but feel free to move it out to where it fits best. As you migrate
 your entire plugin flora to the new backend system, you will probably make more
 and more of these modules as "first class" things, living right next to the
 implementations that they represent, and being exported from there.
+
+### The Auth Plugin
+
+A basic installation of the auth plugin will look as follows.
```

Comment:
Maybe clarify upfront that what's coming is an example specifically for microsoft but that we'll show further down how to adapt it to your particular provider. That's still probably less confusing than for example using a fake provider name or whatnot in this particular section